core/templates/components/forms/custom-forms-directives/thumbnail-display.component.ts

```diff
@@ -45,8 +45,8 @@ export class ThumbnailDisplayComponent implements OnInit, OnChanges {
   updateSvgInViewIfSafe(): void {
     // If the SVG image is passed as base64 data.
     if (this.imgSrc.indexOf('data:image/svg+xml;base64') === 0) {
-      const safeResourceUrl = this.svgSanitizerService.getTrustedSvgResourceUrl(
-        this.imgSrc);
+      const safeResourceUrl = this.svgSanitizerService
```

Comment:
Can we undo this? Seems like nothing but the formatting has changed.

Reply:
Done